configs/raspberrypi4

```diff
@@ -0,0 +1,8 @@
+BR2_arm1176jzf_s=n
+BR2_cortex_a72=y
+BR2_ARM_FPU_NEON_VFPV4=y
+BR2_LINUX_KERNEL_INTREE_DTS_NAME="bcm2711-rpi-4-b"
+BR2_PACKAGE_RPI_FIRMWARE_VARIANT_PI4=y
+BR2_KERNEL_HEADERS_5_4=n
+BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_5_4=y
```

Comment:
Why are you removing the 5.4 kernel headers, and then adding custom 5.4 headers below?

Reply:
I think the PR author used buildroot 2020-3. It does add all the weird stuff to the defconfig.